test/cmdlineTests/combined_json_with_devdoc/c.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+
+contract C {}
```

Comment:
I'd just put this directly in `input.sol` to simplify the test.

Also, would be nice to have some actual dev doc here. For example:
```suggestion
/// @dev This is devdoc.
contract C {}

/// And this is a notice.
contract D {}
```